pyiron_base/project/archiving/export_archive.py

```diff
@@ -4,15 +4,14 @@
 import tempfile
 from typing import Optional
 
-from pyiron_base.project.archiving.shared import getdir
-
 
 def copy_files_to_archive(
     directory_to_transfer: str,
     archive_directory: str,
     compress: bool = True,
     copy_all_files: bool = False,
     arcname: Optional[str] = None,
+    df: Optional["DataFrame"] = None,
```

Comment:
Is there a specific reason to use `"DataFrame"` as string rather than importing the class from `pandas`?

Reply:
Because `pandas` is not imported in this file.

Reply:
Or actually I just assumed that it's better to not import extra classes just for type hinting.